novice-py-getting-started.Rmd

```diff
@@ -0,0 +1,335 @@
+# Getting Started with Python
+
+As stated in the intro, our overall goal is to work with people, programs 
+and data. In this section, we will focus on programs and data as we learn 
+how to run Python code and  
+how data is stored and accessed on a computer. 
+
+## Introduction to Spyder
+
+Starting with programs - over the entirety of this book, we'll be writing 
+programs (or, in verb form, *programming*) in order to accomplish our goals 
+of working with data on the computer. 
+Programming is one way to make a computer do something for us. 
+Instead of clicking, we'll mostly be typing; instead of doing 
+what someone else has pre-defined, we'll have a lot of flexibility to 
+do what we want. 
+
+Just like using a web browser to access websites, and a program like 
+Word to write documents, it's helpful to have a program on your computer 
+that is designed to make it easy to write and run code. This kind of program 
+is called an "IDE" or Integrated Development Environment. 
+
+The one we'll be using in this book for writing Python code is called Spyder.
+Spyder (like many IDEs) opens up and has many panes, that each have a different 
+purpose. 
```

Comment:
screenshot? and say more strongly that we will introduce the panes one by one, so don't be frightened

Reply:
Yes, either screenshot or schematic. Added as TODO.